src/test/java/com/google/step/youtube/DescriptionParserTest.java

```diff
@@ -1,12 +1,200 @@
 package com.google.step.youtube;
 
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-// @RunWith(JUnit4.class)
+@RunWith(JUnit4.class)
 public final class DescriptionParserTest {
 
+    /*                                                                   *
+     * ==================== GENERAL TESTS FOR PARSE ==================== *
+     *                                                                   */
+
+    /** String will return no matches from all regexs in parse() */
+    @Test
+    public void parse_matchNone() {
+        String desc = "Check out our UPDATED version which has all the NEW ELEMENTS here:  https://youtu.be/rz4Dd1I_fX0"
+                        + "\nThe TEETH Song (Memorize Every Tooth): https://youtu.be/PI3hne8C8rU"
+                        + "\nDownload on ITUNES: http://bit.ly/12AeW99 ";
+
+        Assert.assertEquals(Collections.emptyList(), DescriptionParser.parse(desc));
```

Comment:
Using assertThat(expected, equalTo(actual)) is generally better practice since it's more readable. You'll also want to have the assert methods as static imports to have it be more concise.

Same applies for all asserts below.

Reply:
Changed to use Hamcrest assert library for assertThat and equalTo. Also added static imports.